1209/DEDA/index.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: pid
+title: Famiclone and Sega Mega Drive controller to USB adapters
+owner: Cluster
+license: GPLv3
+site: https://github.com/ClusterM/nessmd2usb
+source: https://github.com/ClusterM/nessmd2usb
```

Comment:
What file is for the schematic? I only see `.dip` which is just PCB from what I can tell.

Reply:
Oops, my mistake.

Reply:
Fixed. I commited totally new version based on STM32. Schematic also added.